backend/controllers/follows.js

```diff
@@ -0,0 +1,33 @@
+const Follows = require("./../models/Follows");
+
+module.exports.createFollows = async (req, res) => {
+  try {
+    // Add tag to Follows model
+    console.log("creating follows")
+    const follows = await Follows.create(req.body);
+
+    return res.status(200).json({
+      data: { follows: { id: follows.id } },
+      message: "Follows created.",
+    });
+  } catch (err) {
+    console.log("#####" + err);
+    return res.status(400).json({
+      message: "Error while creating Follows.",
+    });
+  }
+};
+
+module.exports.getFollows = async (req, res) => {
+  try {
+    const follows = await Follows.findOne({ where: { id: req.query.id } });
```

Comment:
This should be a find query right?
and we should be querying by userId
Because we want all followers for that userId?

Reply:
yes, the actual API will use userId. This was a supporting API to test the model